Author: <jari.aalto@cante.net>
Description: daa
Author: Jari Aalto <jari.aalto@cante.net>
src/ltdl.cpp: Fix Gcc 4.4 error: invalid conversion from const char* to char*
--- a/src/ltdl.cpp
+++ b/src/ltdl.cpp
@@ -3361,7 +3361,7 @@
 {
   lt_dlhandle	handle		= 0;
   char *	tmp		= 0;
-  char *	ext		= 0;
+  const char *	ext		= 0;
   size_t	len;
   int		errors		= 0;
 
